### PR TITLE
Create create org

### PR DESCRIPTION
**Pull request by adarshmgit**
i have added new script, please review and merge to master

### CODE REVIEW

**Comment by asdm96** [comment]
i have added reviews please check and revert back

File: create org [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class A{
```
Comment (asdm96) [inline create org:1]: please add java docs

Comment (asdm96) [inline create org:1]: add java doc

**Review by asdm96 — APPROVED** [review]

File: create org (hunk 1 above)
Comment (adarshmgit) [reply create org:1]: i have added please check it

**Pull request merged**